Gini-iOS-SDK/GINIPartialDocumentInfo.m

```diff
@@ -10,7 +10,7 @@
 
 @implementation GINIPartialDocumentInfo
 
-- (instancetype)initWithDocumentUrl:(NSString *)documentUrl rotationDelta:(int)rotationDelta {
+- (instancetype)initWithDocumentUrl:(NSString * _Nullable)documentUrl rotationDelta:(int)rotationDelta {
```

Comment:
Would initializing a document info without an URL be a valid case?
If it's only because the property is nullable, I'd say that the init would still be non nullable.

Reply:
Yes, doesn't make sense. I revert it back to the previous state.

Reply:
Done!